tlsconfig/config.go

```diff
@@ -83,6 +84,22 @@ func certPool(caFile string) (*x509.CertPool, error) {
 	return certPool, nil
 }
 
+// minVersion returns the minimum TLS version matching 'version', the MinVersion string.
+func minVersion(version string) (uint16, error) {
+	allVersions := map[string]uint16{
+		"VersionSSL30": tls.VersionSSL30,
```

Comment:
@cyli I am not convinced we should reuse the same string as the Go constants. If anything I would remove the `Version` prefix but `SSL30`, `TLS10`, `TLS11` and `TLS12` do not seem to be something very widespread. Maybe `SSL3.0`, or `SSL 3.0` ? I couldn't find in the RFC any standard string for this.